### ISSUE TITLE
Show Twitter videos on tweets

### ISSUE THREAD

**Issue by pluja** [pluja/Parasitter#32]
For now, the only attached media that is available on Tweets are images. It is needed that the user could play media such as videos too.

**Comment by Logan1x** [comment]
Yes and if you look at nitter, you can play videos in nitter. So why aren't these visible here?

**Comment by pluja** [comment]
Because it is still not implemented
